eng/emitter-package.json

```diff
@@ -1,11 +1,12 @@
 {
   "main": "dist/src/index.js",
   "dependencies": {
-    "@azure-tools/typespec-rust": "0.3.0"
+    "@azure-tools/typespec-rust": "0.6.0"
```

Comment:
`v0.7.0` is now available and it fixes the Clippy (and other) issues once you migrate to the latest tsp. You'll also need to migrate the below to `0.50.0` and `0.64.0` respectively.

Reply:
Great, thanks! Regenerated!